cves/kernel/CVE-2016-0823.yml

```diff
@@ -186,8 +191,8 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  note:
-  answer:
+  note: RFC spec. Refered to as such in a mailing list.
```

Comment:
More explanation here would be nice. What evidence was given for this violation of the RFC spec in the mailing list? A link to the mail entry might be nice too, if that is something we are doing in these entries